search.md

```diff
@@ -1,8 +0,0 @@
----
-layout: standalone
-title: Search
----
-
-<p class="text-muted">Search everything on this site.</p>
-
-{% include search_box.html %}
```

Comment:
i think this is still worth keeping, even if not prominently linked in the nav bar.

but, not a big deal

Reply:
Brought it back, linked in bar with icon. And removed search from archive, since now we have this back.